seqr/views/apis/awesomebar_api_tests.py

```diff
@@ -152,6 +152,7 @@ class LocalAwesomebarAPITest(AuthenticationTestCase, AwesomebarAPITest):
 class AnvilAwesomebarAPITest(AnvilAuthenticationTestCase, AwesomebarAPITest):
     fixtures = ['users', 'social_auth', '1kg_project', 'reference_data']
 
+    @mock.patch('seqr.views.utils.terra_api_utils.SOCIAL_AUTH_PROVIDER', TEST_OAUTH2_PROVIDER)
```

Comment:
instead of updating this for every test case,  we should update the `AnvilAuthenticationTestCase` to include this mock